code/src/components/Item.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import { todos } from 'reducers/todos';
+
+const ItemContainer = styled.li`
+  display:flex;
+  padding: 8px;
+  margin: 8px 0;
+  font-size: 24px;
+  font-family: Arial, Helvetica, sans-serif;
+  width: 80%;
+  color: #3f3f3f;
+  background-color:#e5c5b5;
+  list-style: none;
+  text-align: left;
+  display: flex;
+  justify-content: space-between;
+`;
+const ListRemoveButton = styled.button`
+  font-size: 3vw;
+  border:none;
+  color: #3f3f3f;
+  background-color:#e5c5b5;
+`;
+const ListCompleteButton = styled.button`
+  font-size: 3vw;
+  color: #3f3f3f;
+  border:none;
+  background-color:#e5c5b5;
+`;
+const Text = styled.text`
+align-self: center;
+`;
+
+export const Item = ({ id,text,isComplete }) => {
+  const dispatch = useDispatch();
+
+  const onCompleteChange = () => {
+    dispatch(todos.actions.toggleComplete(id));
+  };
+  const removeTask = () => {
+    dispatch(todos.actions.removeOne(id));
+  };
+
+  return (
+    <>
+      <ItemContainer>
+        <ListCompleteButton onClick={onCompleteChange}>
+          {isComplete ? "💃" : "🙀"}
+        </ListCompleteButton>
+        <Text>{text}</Text> 
+        <ListRemoveButton onClick={removeTask}>
+        ⓧ
+        </ListRemoveButton>
+      </ItemContainer>
+    </>
+  )
+};
```

Comment:
I so love the creativity with the emoji buttons!!😍 Something that I always like to do with form-type projects is to disable the possibility for the user to enter empty spaces as a message, that could be done with a little function like this one:
` const emptyTask = (value) => value.replace(/\s/g, "").length === 0;

 <TaskButton
          type="submit"
          disabled={
            newTask.length < 5 || newTask.length > 140 || emptyTask(newTask)
          }
        >
          {" "}
          +{" "}
        </TaskButton>`

Absolutely not a necessity tho, only if you feel for implementing it, of course!